src/main/resources/migration/common/V17__rename_user_input_tables.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE calculation_request_user_input
```

Comment:
Anyone know the best practice for testing a migration script like this? I could take a database dump from dev and test it runs as expected.

Reply:
I was wondering if this could be done slightly differently.

Perhaps leave the original table in place and do a `create tables as select ....`

Release it, if it works add another migration for cleaning up the old table and then release that.

Just a thought.

Reply:
I think the only potential issue is dropping the column. The other changes don't result in any loss of data.

I think I could leave column there, do the release to dev and test it. Then only cleanup the column once confident.

Reply:
Sounds reasonable to me.